### DISCUSSION TITLE
FR: Favourite or save Plugins in plugin store

### DISCUSSION THREAD

**Discussion by CreateSean** [craftcms/console#73] [Plugin Store]
### Description
At the full plugin store https://plugins.craftcms.com I would like to be able to favorite or save plugins so I can easily find them later.

Many times I see a new plugin come in and think that would be awesome, but I don't need it now. Then  I forget about the plugin. Being able to easily find plugins I've saved later would be very helpful.

### Steps to reproduce

1.
2.

### Additional info

- Craft version:
- PHP version:
- Database driver & version:
- Plugins & versions:

**Comment by ryanmasuga** [comment]
I've thought the same thing. The store is only going to grow, so it will be harder to remember those unique ones that do some niche thing and that you swear you'd remember the name of but then can't find when you actually have a need for it (e.g. it has some ridiculous "clever" name that doesn't help in a search, or the description/searchable text is horrible).

**Comment by outline4** [comment]
**Yes please!** 
Favs!  
Lists!  
Star ratings!  
And (maybe) being able to bulk purchase/install a list of plugins.

**Comment by CreateSean** [comment]
Still would like to see this. Again today I discovered a plugin that I I know I would like to save for future use, but am unable to do so.

**Comment by khalwat** [comment]
Example: https://twitter.com/eecms/status/1400446856392499201?s=09

**Comment by CreateSean** [comment]
https://twitter.com/eecms/status/1400446856392499201?s=09

It's been over 2 years and no sign of this. But now the ExpressionEngine store has it. 

Hopefully soon.

**Comment by tsawitzki** [comment]
+1

definitely have the same experience with the "cheer now, forget later" store experience with awesome looking new plugins or random finds. shouldn't be over-engineered or too complex of course, however a big bonus could be to also have favorite lists for certain purposes or clients, e.g. "APIs related", "Authoring Experience", etc.
